### PR TITLE
Upate main README

### PR DESCRIPTION
**Pull request by Olshansk**
- Update Pocket Link
- Add a ToC
- Remove duplicate headings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,29 +164,20 @@ Awesome DeFi apps you can deploy on Akash
...
 - [Ethereum 2.0](Ethereum_2.0)
-- [POKT Network](pokt_network)
+- [POKT Network](pokt-network)
 - [Polkadot](polkadot)
 - [Kadena](Kadena)
 - [Bitcoin Cash Node](bitcoincashnode)
 - [Handshake](handshake)
 - [Fuse Network Node](fuse-network-node)
 
-### Blogging
```
Comment (arno01) [inline README.md:147]: please keep the blogging section

File: README.md [hunk 2]
```diff
@@ -7,7 +7,38 @@ Instructions on how to deploy the SDL files in this repository can be found [her
...
+- [Databases and Administration](#databases-and-administration)
+- [DeFi](#defi)
+- [Benchmarking](#benchmarking)
+- [Blockchain](#blockchain)
+- [Business](#business)
+- [Games](#games)
+- [Game Servers](#game-servers)
+- [Hosting](#hosting)
+- [Media](#media)
+- [Search Engines](#search-engines)
```
Comment (arno01) [inline README.md:29]: I think these two can be combined

**Review by arno01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Olshansk) [reply README.md:147]: It is a copy-paste of line 80

Screenshot 2023-08-02 at 1 41 02 PM

Comment (okdas) [reply README.md:147]: @arno01 this appears to be a duplicate; hence it was deleted.

You can actually see “Blogging” appear twice on the `master` branch currently.

Is there anything else blocking this PR?

Thanks!

**Comment by Olshansk** [comment]
@arno01 Could you re-review when you have a chance?

**Comment by Olshansk** [comment]
@arno01 Wanted to bump this PR

**Pull request merged**